tests/test_events.py

```diff
@@ -0,0 +1,115 @@
+# SPDX-License-Identifier: MIT
+
+from typing import Any
+
+import pytest
+
+import disnake
+from disnake import Event
+from disnake.ext import commands
+
+
+@pytest.fixture
+def client():
+    return disnake.Client()
+
+
+@pytest.fixture
+def bot():
+    return commands.Bot(
+        command_prefix=commands.when_mentioned,
+        command_sync_flags=commands.CommandSyncFlags.none(),
+    )
```

Comment:
should these be part of conftest.py? Its up to you, really.

Reply:
That would make sense once we set up more comprehensive tests and proper state/client mocks, but for now it's simple enough to keep in this module.